src/coreclr/jit/importer_vectorization.cpp

```diff
@@ -6,30 +6,105 @@
 #pragma hdrstop
 #endif
 
+// Mirrors StringComparer.cs
```

Comment:
StringComparison.cs?

Can you add a comment to `StringComparison.cs`  that the values of the `StringComparison` enum are "intrinsically" known by the jit and can't be changed on their own?

Reply:
I actually did it in https://github.com/dotnet/runtime/pull/65288#discussion_r805992952 but Jan suggested to remove it because there is no way those will ever change (set in stone basically)

Reply:
I see. Well, ok then.